scripts/clean

```diff
@@ -43,4 +43,26 @@ if [ ! -z "$1" ]; then
     fi
   done
   rm -f $STAMPS/$1/build_*
+}
+
+if [ "$1" = "--all" ]; then
+  if [ ! -z "$2" ]; then
+    for PROJECT in $(ls -1 projects); do
+      for archfile in projects/$PROJECT/linux/linux.*.conf ; do
+        if [ ! -f "$archfile" ]; then
+          for archfile in projects/$PROJECT/linux/*/linux.*.conf ; do
+            archfile=$archfile
+          done
+        fi
+        echo $archfile
```

Comment:
Maybe you're only interested in the last (our only one of) the versioned linux directories that may exist, in which case the inner `for` loop makes more sense.